resources/qml/PrintSetupHeaderButton.qml

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2021 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.1
+import QtQuick.Controls 2.1
+
+import Cura 1.0 as Cura
+import UM 1.5 as UM
+
+ToolButton
+{
+    id: base
+
+    property alias tooltip: tooltip.text
+
+    contentItem: Label {}
+
+    Cura.ToolTip
+    {
+        id: tooltip
+        visible: base.hovered
+        targetPoint: Qt.point(parent.x, Math.round(parent.y + parent.height / 2))
+    }
+
+    background: Rectangle
+    {
+        color:
+        {
+            if(base.enabled)
+            {
+                if(base.valueError)
+                {
+                    return UM.Theme.getColor("setting_validation_error_background");
+                }
+                else if(base.valueWarning)
+                {
+                    return UM.Theme.getColor("setting_validation_warning_background");
+                }
+                else
+                {
+                    return UM.Theme.getColor("setting_control");
+                }
+            }
+            else
+            {
+                return UM.Theme.getColor("setting_control_disabled");
+            }
+        }
+
+        radius: UM.Theme.getSize("setting_control_radius").width
+        border.width: UM.Theme.getSize("default_lining").width
+        border.color:
+        {
+            if (base.enabled)
+            {
+                if (base.valueError)
+                {
+                    return UM.Theme.getColor("setting_validation_error");
+                }
+                else if (base.valueWarning)
+                {
+                    return UM.Theme.getColor("setting_validation_warning");
+                }
+                else if (base.hovered)
+                {
+                    return UM.Theme.getColor("setting_control_border_highlight");
+                }
+                else
+                {
+                    return UM.Theme.getColor("setting_control_border");
+                }
+            }
+            else
+            {
+                return UM.Theme.getColor("setting_control_disabled_border");
+            }
+        }
+        UM.RecolorImage
+        {
+            id: downArrow
+            anchors.verticalCenter: parent.verticalCenter
+            anchors.right: parent.right
+            anchors.rightMargin: UM.Theme.getSize("default_margin").width
+            width: UM.Theme.getSize("standard_arrow").width
+            height: UM.Theme.getSize("standard_arrow").height
+            sourceSize.height: width
+            color: base.enabled ? UM.Theme.getColor("setting_control_button") : UM.Theme.getColor("setting_category_disabled_text")
+            source: UM.Theme.getIcon("ChevronSingleDown")
+        }
+        Label
```

Comment:
Change to UM label (and check what properties no longe rneed to be reset because of that change.

Reply:
https://github.com/Ultimaker/Cura/pull/11336/commits/200e4df0254a5a3f4907331da6ee82a3030ba575

Reply:
https://github.com/Ultimaker/Cura/pull/11336/commits/b3a6d8a1fc4b032a5816d51d030646ab46bc5766
https://github.com/Ultimaker/Cura/pull/11336/commits/a66275ce73c0e0e1f8caf37c3c4f3d2a266dfe34